brackend/db/models.py

```diff
@@ -45,6 +58,42 @@ class UserTournament(Base):
     id = Column(Integer, primary_key=True)
     user_id = Column(Integer, ForeignKey("users.id"))
     tournament_id = Column(Integer, ForeignKey("tournaments.id"))
+    is_organizer = Column(Boolean, default=False)
+
+
+class Bracket(Base):
+    """Table representing a bracket."""
+
+    __tablename__ = "brackets"
+    id = Column(Integer, primary_key=True)
+    name = Column(String(255), nullable=False)
+    tournament = Column(Integer, ForeignKey("tournaments.id"))
+    rounds = relationship("Round", backref="brackets")
+
+
+class Round(Base):
+    __tablename__ = "rounds"
+    id = Column(Integer, primary_key=True, autoincrement=False)
+    name = Column(String(255), nullable=False)
+    bracket = Column(Integer, ForeignKey("tournaments.id"))
+    matches = relationship("Match", backref="rounds")
+
+
+class Match(Base):
+    __tablename__ = "matches"
+    id = Column(Integer, primary_key=True)
```

Comment:
Is it too late for me to bring up whether we should be using UUIDs instead of integers for ids? I've never used Integer/incremental IDs in a production system before. But maybe there's an argument for sticking with integer id's as we shouldn't really be thinking of this as a production system until we actually need to?